### DISCUSSION TITLE
Too many line breaks when using ::

### DISCUSSION THREAD

**Discussion by Zouxxyy** [scalameta/scalafmt#3746] [Q&A]
## Configuration (required) ##

Please paste the contents of your `.scalafmt.conf` file here:
```
version = 3.4.3
```

full conf, see https://github.com/apache/incubator-paimon/blob/master/.scalafmt.conf

```
spotless.version=2.13.0
```

## Command-line parameters (required) ##

```shell
mvn spotless:apply
```

## Steps

Given code like this:
```scala
val expertResult2 =
    Row("+I", 1, "v_1") :: Row("-U", 1, "v_1") :: Row("+U", 1, "v_1_new") :: Row("+I", 2, "v_2") :: 
       Row("-U", 2, "v_2") :: Row("+U", 2, "v_2_new") :: Row("+I", 3, "v_3") :: Nil
```

## Problem

Scalafmt formats code like this:
```scala
val expertResult2 =
    Row("+I", 1, "v_1") :: Row("-U", 1, "v_1") :: Row("+U", 1, "v_1_new") :: Row(
        "+I",
         2,
         "v_2") :: Row("-U", 2, "v_2") :: Row("+U", 2, "v_2_new") :: Row("+I", 3, "v_3") :: Nil
```

## Expectation

The code after format is a bit strange, don't want see too many line breaks, is there any relevant configuration that can improve it, thanks

**Comment by kitbellew** [comment]
have you had a chance to read through the configuration document online? https://scalameta.org/scalafmt/docs/configuration.html
